jekyll/_cci2/first-steps.adoc

```diff
@@ -10,62 +10,101 @@ contentTags:
 :icons: font
 :experimental:
 
-To run your very first build on CircleCI, go to the link:https://circleci.com/signup/[Sign Up] page. Sign up with your GitHub, Bitbucket or GitLab account, or your email address for the option to connect to your code later in the process.
+To start building on CircleCI, go to the link:https://circleci.com/signup/[Sign Up] page. Sign up with your email address and connect your GitHub, Bitbucket, or GitLab account in the process.
 
-[#vcs-signup]
-== Sign up with GitHub or Bitbucket
+- If you are **creating a new CircleCI org** for your team, follow the xref:#sign-up-and-create-an-org[Sign up and create an org] steps below. 
+- If you are signing up for a CircleCI account through an **invite** from someone on your team, follow the steps in the xref:#sign-up-with-an-invite[Sign up with an invite] section.
 
-. Click on either link:https://circleci.com/auth/vcs-connect?connection=Github[**Sign Up with GitHub**] or link:https://circleci.com/auth/vcs-connect?connection=Bitbucket[**Sign Up with Bitbucket**] to start the authentication process and allow CircleCI to access your code.
+[#vcs-integrations]
+== VCS integrations
+
+CircleCI supports GitHub, Bitbucket, and GitLab projects. If you use **GitHub** or **GitLab**, and are creating a CircleCI organization for your team, you will set the version control system (VCS) integration type and authenticate with your VCS provider during the signup process. Team members that you invite afterwards to your CircleCI org can join in CircleCI without also authenticating with your VCS. However, if they are to create new projects, they will need to authenticate during the project creation process.
+
+If you use **Bitbucket**, every team member signing up for a new account and joining your org will authenticate with Bitbucket _during_ the signup process.
+
+[#sign-up-and-create-an-org]
+== Sign up and create an org
+
+If you are new to CircleCI, you will create a CircleCI organization. A CircleCI organization may correspond directly to a _personal_ CircleCI account, or more commonly, to a _shared_ account in which a team collaborates and with which multiple team members' personal CircleCI accounts can be associated. 
+
+. To create an account for free with CircleCI, enter your email address and password on the link:https://circleci.com/signup/[Sign Up] page.
+
+. On the next screen, respond to prompts that best describe your role and your engineering organization.
+
+. Then, connect your code. Choose the VCS that contains your code repositories: GitHub, GitLab.com (cloud), GitLab Self-Managed, or Bitbucket.
+
+[#GitHub]
+=== GitHub
```

Comment:
I reckon a banner here for rollout week would be good, maybe we can say some users may have an experience more in line with Bitbucket for this time?

Reply:
With a link to the GitHub OAuth integration page

Reply:
@rosieyohannan A banner could be good. I think the other option would be to go live with this page only at or close to 100% rollout.

I'm leaning a bit more towards the banner